infrastructure-scripts/claude_start.sh

```
@@ -0,0 +1,1640 @@
+            echo -e "${BLUE}🤖 Qwen mode selected - using Qwen3-Coder via vast.ai${NC}"
+            echo -e "${BLUE}💡 Features: Remote Qwen3-Coder model via vast.ai GPU instances${NC}"
+            echo -e "${BLUE}💡 Model: qwen3-coder (optimized for coding tasks)${NC}"
+            echo ""
+
+            # Check if API proxy is available
+            # TERMINAL SESSION PRESERVATION: Never exit terminal on errors - let users Ctrl+C to go back
+            API_PROXY_PATH="$HOME/projects/claude_llm_proxy/api_proxy.py"
+            if [ ! -f "$API_PROXY_PATH" ]; then
+                echo -e "${RED}❌ LLM self-hosting repository not found${NC}"
+                echo -e "${BLUE}💡 LLM proxy (for Qwen and Cerebras) is maintained in the separate claude_llm_proxy repository${NC}"
+                echo -e "${BLUE}💡 Clone it with: cd ~/projects && git clone https://github.com/$USERorg/claude_llm_proxy.git${NC}"
```

Shellcheck (0.10.0)

[warning] USERorg is referenced but not assigned.

(SC2154)